routers/api/v1/api.go

```diff
@@ -513,12 +513,20 @@ func RegisterRoutes(m *macaron.Macaron) {
 		// Users
 		m.Group("/users", func() {
 			m.Get("/search", user.Search)
-
 			m.Group("/:username", func() {
 				m.Get("", user.GetInfo)
+				m.Get("/followers", user.ListFollowers)
+				m.Group("/following", func() {
+					m.Get("", user.ListFollowing)
+					m.Get("/:target", user.CheckFollowing)
+				})
+				m.Get("/gpg_keys", user.ListGPGKeys)
 				m.Get("/heatmap", mustEnableUserHeatmap, user.GetUserHeatmapData)
-
+				m.Get("/keys", user.ListPublicKeys)
+				m.Get("/orgs", org.ListUserOrgs)
 				m.Get("/repos", user.ListUserRepos)
+				m.Get("/starred", user.GetStarredRepos)
+				m.Get("/subscriptions", user.GetWatchedRepos)
```

Comment:
I wonder if all this info is public for all users?

Reply:
Yes, GitHub makes that public for all users too. See https://api.github.com/users/richmahn, https://api.github.com/users/richmahn/favorites, https://api.github.com/users/richmahn/gpg_keys, https://api.github.com/users/richmahn/keys, etc. These are public keys which a client may need to see/use.

Reply:
I even searched to see why these were required the user to be authenticated, yet never found a reason. I assume it was just done that way because Gogs originally was more for just a single or few users.